installer/pkg/components/alertmanager/constants.go

```diff
@@ -0,0 +1,10 @@
+package alertmanager
+
+const (
+	Name      = "main"
```

Comment:
Same question as https://github.com/gitpod-io/observability/pull/203#discussion_r941043351 ☺️

Reply:
That's the name that comes from upstream, I'm just copy-pasting 😅. The Alertmanager CR will be named `main`, then the statefulset created by the operator will be named `alertmanager-main`.

You do have a good point that all the resources that are not created by the operator will get awkward name... let me fix that